### PR TITLE
Create tabsiforu - Wooden Barricade.yml

### PR DESCRIPTION
**Pull request by ghost**
see https://github.com/HippieStationCode/HippieStation13/pull/2825

### CODE REVIEW

File: html/changelogs/tabsiforu - Wooden Barricade.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: tabsiforu
+
+delete-after: True
+
+- tweak: "Wooden barricade costs 6 wood planks to build, up from 5, in order to be consistent with spritework."
```
Comment (ghost) [inline html/changelogs/tabsiforu - Wooden Barricade.yml]: You need to put 2 spaces before the minus sign.

**Comment by ghost** [comment]
@McBawbaggings space created

**Comment by ghost** [comment]
It needs to before it, not after it friendo.

**Comment by danhibiki** [comment]
Just add your changelog addition to your barricade update pull, if its accepted it will be accepted with the log, instead of having two seperate pulls for it.

**Comment by ghost** [comment]
@danhibiki i will do this next time though as for now i would like to nuke my fork after this is pulled and start over

**Comment by ghost** [comment]
add a single space between the minus and tweak to see if travis likes it?

File: html/changelogs/tabsiforu - Wooden Barricade.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author: tabsiforu
+
+delete-after: True
+
+  -tweak: "Wooden barricade costs 6 wood planks to build, up from 5, in order to be consistent with spritework."
```
Comment (JohnGinnane) [inline html/changelogs/tabsiforu - Wooden Barricade.yml:5]: Format like this:

```
changes:
 - tweak: "Changed..."
```

See: https://github.com/HippieStationCode/HippieStation13/pull/2833/files#diff-039ebccff48fd97e0a6f7d175430691fR14

**Comment by ghost** [comment]
fuck it i'm making one merge

**Pull request closed without merging**